grobid_client.py

```diff
@@ -5,7 +5,7 @@
 import argparse
 import time
 import concurrent.futures
-from client import ApiClient
```

Comment:
I wonder why do you need the root directory in this import?

Reply:
It might not be necessary - I'm not really a python pro. However my understanding is that:
`from client import ApiClient` is a relative import. It relies on the fact that the code gets called from within the grobid_python_client folder to work. However, when I as a user now import the grobid-client, it runs the import statement from my current package (which most likely is my main package) which in turn will fail. 

Using absolute imports specifying the package name should work regardless of the location it is called. I'm not 100% sure, but looking at other popular open source python packages ([pandas](https://github.com/pandas-dev/pandas/blob/b3ed7f9873f7f4689aa8266bf0ed77912d6d1b9b/pandas/core/frame.py#L80), [matplotlib](https://github.com/matplotlib/matplotlib/blob/440af795b3745a0d95f892e72adc45b0b4e7465d/lib/matplotlib/axis.py#L19)) this seems to be the normal behavior. I think another solution would be to use explicit relative imports like [tqdm does](url) (`from .client import ApiClient`), however this should be tested again since tqdm seems to have a lot of meta-code in their module files.

I think [this guide](https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html#absolute-vs-relative-import) from Chris Yeh also helped my understanding.

Reply:
Yet another option would be to import the classes in the `__init__.py` file, since this will surely lie within the grobid-client. However given the examples from pandas and matplotlib this seems to be an unusual way and would also mean importing all possible helper classes of the package, regardless of what the user actually needs just so it's available if the grobid-package needs it.